fullcontact/schema/person_schema.py

```diff
@@ -49,9 +56,8 @@ class ProfileSchema(BaseCombinationSchema):
     )
 
 
-class PersonSchema(BaseSchema):
-    schema_name = "Person"
-
+class PersonSummarySchema(BaseSchema):
```

Comment:
This is the schema for the input object correct? PersonSummary is also something else coming back in the response -- the top several fields before details: `https://dashboard.fullcontact.com/api-ref#person-enrichment`.

Perhaps we should call this something like "MultiFieldRequestSchema" or even "PersonRequestSchema"?  Multi Field Request summary is more accurate.

Reply:
Done. Renamed.